backend/app/controllers/spree/admin/base_controller.rb

```diff
@@ -42,7 +42,7 @@ def check_alerts
         end
 
         def should_check_alerts?
-          return false if !Rails.env.production? || !Spree::Config[:check_for_spree_alerts]
+          return false if !Spree::Config[:check_for_spree_alerts]
```

Comment:
I'd prefer this was reverted back we don't want everyones development machines pinging the alerts server.